libpod/healthcheck_supported.go

```diff
@@ -0,0 +1,69 @@
+// +build linux darwin
```

Comment:
OS X doesn't have systemd - this might build there, but it won't work there. I'd just make this healthcheck_linux.go and drop Darwin.

Reply:
@mheon the real issue here is why are we dragging in healthcheck backend to our cli.  I'm inclined to leave things as is and dive into that next week.